var/spack/repos/builtin/packages/root/package.py

```diff
@@ -164,29 +162,31 @@ class Root(CMakePackage):
             multi=False,
             description='Use the specified C++ standard when building.')
 
-    # ###################### Dependencies ##############################
+    # ###################### Dependencies ######################
 
-    # minimum cmake version required
     depends_on('cmake@3.4.3:', type='build')
     depends_on('pkgconfig', type='build')
 
-    depends_on('lz4',    when='@6.13.02:')
-    depends_on('xxhash', when='@6.13.02:')
-    depends_on('xz')
-    depends_on('pcre')
+    depends_on('blas')
     depends_on('freetype')
+    depends_on('jpeg')
+    depends_on('libice')
     depends_on('libpng')
-    depends_on('ncurses')
```

Comment:
Is `ncurses` completely removed on purpose?

Reply:
Will verify when I get to work.

Reply:
A check of every library and binary fails to find any reference to `ncurses` (or any other kind of curses, for that matter.
```
[greenc@woof] /scratch/greenc/spack-tree-02/install/linux-scientific7-x86_64/gcc-7.3.0/root-6.16.00-byoj3vf6fg3ycjnxq3fpqrhzkza43brh $ for i in lib/*.so bin/*; do file -L $i 2>/dev/null | grep ELF >/dev/null 2>&1 || continue; x=$(ldd $i 2>/dev/null | grep curses 2>/dev/null); [[ -n "$x" ]] && echo "$i: $x"; done
[greenc@woof] /scratch/greenc/spack-tree-02/install/linux-scientific7-x86_64/gcc-7.3.0/root-6.16.00-byoj3vf6fg3ycjnxq3fpqrhzkza43brh $ 
```

Reply:
Checking the root repo, I found [this reference](https://github.com/root-project/root/blob/b5724a3102664d2e130ab165e634dbcf685d6417/interpreter/llvm/src/cmake/config-ix.cmake#L150). If it does not affect to FNAL's use, I would keep it.

Reply:
I have restored the `depends_on('ncurses')` The actual library used is `libtinfo.so` which was confusing, but it is provided by the `ncurses` package.